scripts/eval/yamls/tasks_v0.1.yaml

```diff
@@ -0,0 +1,266 @@
+icl_tasks:
+-
+  label: jeopardy
+  dataset_uri: eval/local_data/world_knowledge/jeopardy_all.jsonl # ADD YOUR OWN DATASET URI
```

Comment:
can you remove all the "add your own comments"? They don't really apply to this version gauntlet file.